Bindings/Python/swig/python_simulation.i

```diff
@@ -68,7 +68,6 @@ MODEL_ADOPT_HELPER(ModelComponent);
 MODEL_ADOPT_HELPER(Body);
 MODEL_ADOPT_HELPER(Probe);
 MODEL_ADOPT_HELPER(Joint);
-MODEL_ADOPT_HELPER(Frame);
```

Comment:
Unclear why these two lines were removed. No explanation found in commit notes or PR comment.